lib/rules/declaration-property-value-no-unknown/__tests__/index.mjs

```diff
@@ -11,6 +11,9 @@ testRule({
 		{
 			code: 'a { top: 0; }',
 		},
+		{
+			code: 'a { @starting-style { opacity: 0; } }',
```

Comment:
minor nit

> - the color property by default
> - the red value by default

Since the rest of the examples don't respect it, you don't have to though.
cf https://stylelint.io/developer-guide/rules#write-tests